spec/helpers/main_helper.rb

```diff
@@ -0,0 +1,36 @@
+module MainHelper
+  def login(email, password)
+    visit '/login'
+
+    fill_in 'spree_user_email', with: email
```

Comment:
You could use here your own method "fill_in_credentials". Don't forget to place that method before login method